great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.navigation-bar {
+	display: flex;
+	padding: 10px;
+	justify-content: center;
+	/*width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;*/
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	background-color: white;
+	width: 100%;
+	align-items: center;
+	height: 36px;
+}
+
+header nav a {
+	vertical-align: middle;
+	margin: 10px 28px;
+	font-size: 15px;
+	text-decoration: none;
+	background-color: white;
+	color: lightgrey;
+}
+
+header nav a:first-of-type {
+	margin-left: 0;
+}
+
+header nav a:hover {
+	color: black;
+	font-size: 18px;
+	border-bottom: 2px dashed black;
+}
+
+
```

Comment:
would like to see more specificity here in your logo, you must always keep in mind that you might end up using another logo tag in a different page and if you're using the same css styling, this could present some issues.